src/components/navigation/document-tab-panel.tsx

```diff
@@ -170,7 +182,7 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
                 tab={subTab.label}
                 section={section}
                 index={index}
-                numOfSections={subTab.sections.length}
+                numSections={subTab.sections.length || 0}
```

Comment:
```suggestion
                numSections={subTab.sections.length}
```
I don't think this is doing anything as length is always a number.